node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/configserver/noderepository/RealNodeRepository.java

```diff
@@ -179,31 +191,24 @@ private static NodeSpec createNodeRepositoryNode(GetNodesResponse.Node node)
                 Optional.ofNullable(node.parentHostname));
     }
 
-    @Override
-    public void updateNodeAttributes(final String hostName, final NodeAttributes nodeAttributes) {
-        UpdateNodeAttributesResponse response = configServerApi.patch(
-                "/nodes/v2/node/" + hostName,
-                new UpdateNodeAttributesRequestBody(nodeAttributes),
-                UpdateNodeAttributesResponse.class);
-
-        if (response.errorCode == null || response.errorCode.isEmpty()) {
-            return;
-        }
-        throw new RuntimeException("Unexpected message " + response.message + " " + response.errorCode);
+    private static NodeRepositoryNode nodeRepositoryNodeFromAddNode(AddNode addNode) {
+        NodeRepositoryNode node = new NodeRepositoryNode();
+        node.openStackId = "fake-" + addNode.hostname;
```

Comment:
Isn't the openstackid supposed to be the instance id in aws?

Reply:
I was hoping this was only needed for tenant hosts since we dont create own certificate for the config container, not sure. Checking with Morten.

Either way, there is no easy way to achieve this here. We would have to set up CKMS and give cfghost permission to talk to AWS... :(

Reply:
Apparently we don't need this, and will just verify the athenz domain and service name only.